code/game/objects/items/weapons/storage/case.dm

```diff
@@ -0,0 +1,71 @@
+/obj/item/weapon/storage/case //created for future tweaks
+	name = "Case"
+	max_w_class = ITEM_SIZE_BULKY
+	w_class = ITEM_SIZE_HUGE
+	var/opened = FALSE //Checking opened case or not
+
+/obj/item/weapon/storage/case/attack_hand(mob/user as mob)
+
+	if ((loc != user) && opened)
+		open(user)
+
+	else
+		close_all()
+		..()
+
+	update_icon()
+	src.add_fingerprint(user)
+	return
+
+/obj/item/weapon/storage/case/open(var/mob/user)
+	if(!opened)
+		return
+	..()
+
+/obj/item/weapon/storage/case/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	update_icon()
+	..()
+
+/obj/item/weapon/storage/case/AltClick()
+	if(!istype(loc, /turf/))
+		to_chat(usr, "\The [src] has to be on a stable surface first!")
+		return
+
+	else
+		opened = !opened
+		anchored = !anchored
+		close_all()
+		update_icon()
+
+/obj/item/weapon/storage/case/update_icon()
+	..()
+
+	icon_state = initial(icon_state)
+	if(!anchored)
+		overlays.Cut()
+		icon_state += "-closed"
+
+/obj/item/weapon/storage/case/donut
+	name = "Special donut delivery case"
+	desc = "Containe a special cooked donut for a connoisseurs of donuts."
```

Comment:
Should be: "Contains specially cooked donuts for donut connoisseurs."

Reply:
Done